packages/core/email/server/src/middlewares/rateLimit.ts

```diff
@@ -0,0 +1,46 @@
+import type { Context, Next } from 'koa';
+import utils from '@strapi/utils';
+import { isString, has, get } from 'lodash/fp';
+import type { Core } from '@strapi/types';
+
+const { RateLimitError } = utils.errors;
+
+export default (config: any, { strapi }: { strapi: Core.Strapi }) =>
+  async (ctx: Context, next: Next) => {
+    const pluginConfig = strapi.config.get('plugin::email') as any;
+    let rateLimitConfig = pluginConfig.ratelimit as any;
+
+    if (!rateLimitConfig) {
+      rateLimitConfig = {
+        enabled: true,
+      };
+    }
+
+    if (!has('enabled', rateLimitConfig)) {
+      rateLimitConfig.enabled = true;
+    }
```

Comment:
```suggestion
    let rateLimitConfig = strapi.config.get('plugin::email.ratelimit') || {};
    rateLimitConfig = {
      enabled: true,
      ...rateLimitConfig,
    };
```
Something like this might work better to set the default